doc/02-Installation.md

```diff
@@ -185,41 +259,36 @@ host  all icingadb 0.0.0.0/0 md5
 host  all icingadb      ::/0 md5
 ```
 
-To apply those changes, run `systemctl reload postgresql`.
+To apply these changes, run `systemctl reload postgresql`.
 
-After creating the database you can import the Icinga DB schema using the
-following command. Enter the password when asked.
+After creating the database, import the Icinga DB schema using the following command:
 
 ```
 psql -U icingadb icingadb < /usr/share/icingadb/schema/pgsql/schema.sql
 ```
+## Configuring Icinga DB
 
-### Running Icinga DB <a id="running-icingadb"></a>
-
-Foreground:
+Icinga DB installs its configuration file to `/etc/icingadb/config.yml`,
+pre-populating most of the settings for a local setup. Before running Icinga DB,
+adjust the Redis and database credentials and, if necessary, the connection configuration.
+The configuration file explains general settings.
+All available settings can be found under [Configuration](03-Configuration.md#Configuration).
 
-```
-icingadb --config /etc/icingadb/config.yml
-```
+## Running Icinga DB
 
-Systemd service:
+The `icingadb` package automatically installs the necessary systemd unit files to run Icinga DB.
+Please run the following command to enable and start its service:
 
+```bash
+systemctl enable --now icingadb
 ```
-systemctl enable icingadb
-systemctl start icingadb
-```
-
-### Enable remote Redis connections <a id="remote-redis"></a>
-
-By default `icingadb-redis` listens only on `127.0.0.1`. If you want to change that, e.g. for Icinga Web 2 or Icinga 2
-running on another node, just change `bind 127.0.0.1 ::1` and `protected-mode yes`
-in `/etc/icingadb-redis/icingadb-redis.conf` to the interface you want to use and to `protected-mode no`.
 
-> WARNING: By default, Redis does not have any authentication that prevents others from accessing it.
-> If you open Redis to an external interface, make sure that you set up appropriate firewall rules or configure TLS
-> with certificate authentication on Redis and its consumers, i.e. Icinga 2, Icinga DB and Icinga Web 2.
+## Installing Icinga Web
 
-### Icinga DB Web
+With Icinga 2 and Icinga DB fully set up, it is now time to
+[install Icinga Web](https://icinga.com/docs/icinga-web-2/latest/doc/02-Installation) and then
+[install Icinga DB Web](https://icinga.com/docs/icinga-db/latest/icinga-db-web/doc/02-Installation).
```

Comment:
Will this link stay valid as we talked about removing that Web sub-tree there?

Reply:
Forgot to adjust that part.